public/app/features/provisioning/components/BulkActions/BulkExportProvisionedResource.tsx

```diff
@@ -0,0 +1,383 @@
+import { css } from '@emotion/css';
```

Comment:
This component needs more thought and refinement, I suggest looking at the bulk move component as an example to make this less complicated.